lib/swaps/Swaps.ts

```diff
@@ -223,12 +191,23 @@ class Swaps extends EventEmitter {
     // generalize the logic below to work for when we are the maker as well
     const { makerCurrency } = Swaps.deriveCurrencies(maker.pairId, maker.isBuy);
     const { makerAmount } = Swaps.calculateSwapAmounts(taker.quantity, maker.price, maker.isBuy);
-    let routes: lndrpc.Route[];
+
+    const swapClient = this.swapClients[makerCurrency];
+    if (!swapClient) throw new Error('swap client not found');
+
+    const peer = this.pool.getPeer(maker.peerPubKey);
+    const destination = peer.getLndPubKey(makerCurrency);
```

Comment:
What do you think about calling this `peer.getPubKey`?

Reply:
I was thinking that this code will need to change, but raiden does not have a pub key. I was thinking of maybe having something like `getIdentifier()` but I wasn't sure so I put it aside for now.

Reply:
`getIdentifier` sounds good, but I suggest to leave it out of this PR.